syntax = "proto3";

package card;

service CardService {
    rpc CreateCard(CreateCardRequest) returns (CreateCardResponse) {}
    rpc GetCards(GetCardsRequest) returns (GetCardsResponse) {}
}

message CreateCardRequest {}

message CreateCardResponse {
    Card card = 1;
}

message Card {
    int32 id = 1;
    string name = 2;
}

message GetCardsRequest {}

message GetCardsResponse {
    repeated Card cards = 1;
}